data/syntax-highlighting/vim/syntax/meson.vim

```diff
@@ -32,8 +32,9 @@ set cpo&vim
 
 " http://mesonbuild.com/Syntax.html
 syn keyword mesonConditional	elif else if endif
-syn keyword mesonRepeat	foreach endforeach
+syn keyword mesonRepeat		foreach endforeach
```

Comment:
What's the purpose of this diff hunk?

Reply:
A tab for editor alignment:
![image](https://user-images.githubusercontent.com/75801/88025795-79d54c00-cb52-11ea-983b-1fd5c4836291.png)

Reply:
I think depends on tab size :)

Reply:
Meh. this file contains `" vim:set sw=2 sts=2 ts=8 noet:` so the change is correct and needed if I open it in vim, but git log seems to align tabs including the leading `-+ ` for the diff context. Sorry...

Reply:
No worries, thanks for eyeballs :)